Housing Unit Inventory (#126)
agrc/metadata-asset-tracking-tool: metadata/planning/housing_unit_inventory_schema.md

# Title

Utah Housing Unit Inventory Field Definitions

## ID

48f9d241-63fb-4ed9-ae35-58d14a3998a6

### TYPE

> General housing type, such as single or multi-family.

Reviewer: ```suggestion
General housing type: single or multi-family.
```

### SUBTYPE

> More detailed housing type classification.

Reviewer: ```suggestion
More detailed housing type classification like single family, townhome, condo, etc.
```

### IS_OUG

> 1 or Yes for features that are owned unit groupings (as opposed to a single-parcel home or apartment)

Reviewer: ```suggestion
1 or Yes for features that are owned unit groupings (usually condos or HOAs grouped with their commonly-owned areas as opposed to a single-parcel home or apartment)
```

### UNIT_COUNT

> The number of dwelling units, based on address point data.

Reviewer: ```suggestion
The number of dwelling units in a property or owned unit grouping
```
